### PR TITLE
Audit LOg message and searching feature in SMP update for snapshot

### PR DESCRIPTION
**Pull request by darshan-clvt**
1. SMP i have Updated "Trusted" Filter fields name as it was not enabled for search casuing issue while search 
2. BE changes For auditlog message "Modify Subscription"

### CODE REVIEW

File: src/api/routes/subscriptions.js [hunk 1]
```diff
@@ -379,9 +379,11 @@ subscriptions.addSubscription = function (app, res, applications, loggedInUserId
                     }
 
 
+                    const actionType = subsCreateInfo.ModifyPlan ? webhooks.ACTION_MODIFY : webhooks.ACTION_ADD;
```
Comment (saichrn7) [inline src/api/routes/subscriptions.js]: what happens if there is no modifyPlan field in subscreateInfo obj ? do you see any exception ?

File: src/api/routes/subscriptions.js [hunk 2]
```diff
@@ -597,17 +599,21 @@ subscriptions.deleteSubscription = function (app, res, applications, loggedInUse
...
-                        entity: webhooks.ENTITY_SUBSCRIPTION,
-                        data: {
-                            subscriptionId: subscriptionId,
-                            applicationId: appId,
-                            apiId: apiId,
-                            userId: loggedInUserId,
-                            auth: subscriptionData.auth
-                        }
-                    });
+                    const actionType = subsCreateInfo.ModifyPlan ? webhooks.ACTION_MODIFY : webhooks.ACTION_DELETE;
```
Comment (saichrn7) [inline src/api/routes/subscriptions.js]: change the ModifyPlan to camelcase pattern

**Review by saichrn7 — CHANGES_REQUESTED** [review]

File: src/api/routes/subscriptions.js (hunk 1 above)
Comment (darshan-clvt) [reply src/api/routes/subscriptions.js]: If no ModifyPlan field it will be in Add subscription .

Comment (darshan-clvt) [reply src/api/routes/subscriptions.js]: No exception

File: src/api/routes/subscriptions.js (hunk 2 above)
Comment (darshan-clvt) [reply src/api/routes/subscriptions.js]: Okay

**Review by saichrn7 — APPROVED** [review]

**Pull request merged**